lib/pymedphys/_experimental/chartchecks/test2.py

```diff
@@ -2,8 +2,14 @@
 
 from pymedphys._experimental.chartchecks.helpers import get_all_dicom_treatment_info
 
-file = "C:/Users/rembishj/Documents/raystation_plan/RP1.2.752.243.1.1.20210123122119948.1140.13064.dcm"
 
-plan = get_all_dicom_treatment_info(file)
+def main():
+    file = "C:/Users/rembishj/Documents/raystation_plan/RP1.2.752.243.1.1.20210123122119948.1140.13064.dcm"
```

Comment:
this test can't pass anywhere but in the specific user's environment.
suggest ensuring that it is anonymised or pseudonymised or has no PHI, and load up to zenodo, and then access via zenodo.

Reply:
This is one of @rembishj's files within experimental. For now, I'd like to let him learn to crawl before running :). It can certainly be on his to-do list, but I'm quite keen for it to not be a blocker for this PR.

The only change this PR has done to that file is move the top-level code to be scoped under `main()`.

Reply:
Please add an issue and ask @rembishj to assign the issue to himself.

The perfect is the enemy of the good, but accumulated cruft is the death of a software package.  So let's make sure that retiring technical debt is a high priority.

Reply:
Yup good call. Will do. See https://github.com/pymedphys/pymedphys/issues/1381

Reply:
And on that note I've also added another issue:

https://github.com/pymedphys/pymedphys/issues/1382